packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx

```diff
@@ -1,16 +1,66 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { translate, Button, Icon } from 'cozy-ui/transpiled/react/'
+import get from 'lodash/get'
 
-const EmptyDocumentHolder = ({ t }) => (
-  <Button theme="ghost" extension="full" align="left">
-    <Icon icon="plus" size={16} className="u-mr-1 u-pa-half" />
-    <span>{t('documents.import')}</span>
-  </Button>
+import { withClient } from 'cozy-client'
+import {
+  translate,
+  ButtonLink,
+  Icon,
+  FileInput
+} from 'cozy-ui/transpiled/react/'
+
+import creditApplicationTemplate from '../../templates/creditApplicationTemplate'
+import DocumentsDataFormContainer from '../../containers/DocumentsDataForm'
+const EmptyDocumentHolder = ({
+  t,
+  documentId,
+  client,
+  linkDocumentSuccess
+}) => (
+  <FileInput
+    onChange={async file => {
+      const dirPath = creditApplicationTemplate.pathToSave
```

Comment:
Would be cool to move this in a method.

Reply:
are we sure we want to upload the docs and create the zip at the same path? Maybe it's ok for now but we could use a `pathToSaveZip` or something

Reply:
ATM we're not sure where we want to save the files and the content. So let's keep it that way and see what happen next, after a real test :)